source/_patterns/01-elements/buttons/02-button-secondary.hbs

```diff
@@ -13,3 +13,11 @@
 <p>
 	{{> elements-button variant='secondary' title='Check' value='loading' state='loading' }}
 </p>
+
+
+<p>
+	<div class="db-bg-success-light-ia">
+		{{> elements-button variant='secondary' value='Secondary' }}
+		{{> elements-button variant='secondary' value='Secondary (disabled)' disabled='true' }}
+	</div>
+</p>
```

Comment:
what about making this a unique pseudo-pattern so that we have a whole area with all of the button variations on this background ?

Reply:
I tried to test this: 
![image](https://user-images.githubusercontent.com/104347736/201329099-397883f7-b38d-4160-97ea-84f81c5e0cb2.png)

I guess we need to change the documentation in the target branch